build.gradle

```diff
@@ -165,6 +165,8 @@ dependencies {
 
     testCompile group: 'junit', name: 'junit', version:'4.12'
     testCompile group: 'xmlunit', name: 'xmlunit', version:'1.3'
+    testCompile group: 'org.hamcrest', name: 'hamcrest-all', version: '1.3'
```

Comment:
I have to admit, I'm not a fan of things just being added to build.gradle without a discussion.  I think we need more broad buy-in that things will be widely used and are value add to the project before they are just adopted